index.html

```diff
@@ -94,169 +95,176 @@ <h3>Tonic</h3>
+                <li class="java">
+                  <img src="./image/html.png" alt="html logo" />
+                  <p>HTML</p>
+                </li>
+                <li class="java">
+                  <img src="./image/css.png" alt="css logo" />
+                  <p>CSS</p>
+                </li>
+            </ul>
+            <ul class="skill">
+              <li class="arro">
+                <p class="frame">Frameworks<li class="fa fa-chevron-right frame-arro"></li></p>
               </li>
-              <li>
-                <a href="#"><img src="./image/vector.png" alt="" /></a>
+              <div class="hr-line"><hr /></div>
+              <li class="arro">
+                <p class="frame">Skill<li class="fa fa-chevron-right frame-arro"></li></p>
               </li>
+              <div class="hr-line last"><hr /></div>
             </ul>
-          </div>
-          <button>Get my resume</button>
-        </div>
-
-        <ul class="language">
-          <li>
-            <ul class="dash">
-              <p class="frame">Languages</p>
-              <li>
-              <span class="fa fa-chevron-down arr-down"></span></li>
-            </ul>
-            <ul class="lang">
-            <li class="java">
-              <img src="./image/js.png" alt="js logo" />
-              <p>Javascript</p>
-            </li>
-            <li class="java">
-              <img src="./image/html.png" alt="html logo" />
-              <p>HTML</p>
-            </li>
-            <li class="java">
-              <img src="./image/css.png" alt="css logo" />
-              <p>CSS</p>
-            </li>
+            </li> 
           </ul>
-          <ul class="skill">
-            <li class="arro">
-              <p class="frame">Frameworks</p>
-              <li class="fa fa-chevron-right frame-arro"></li>
-            </li>
-            <div class="hr-line"><hr /></div>
-            <li class="arro">
-              <p class="frame">Skill</p>
-              <li class="fa fa-chevron-right frame-arro"></li>
-            </li>
-            <div class="hr-line last"><hr /></div>
-          </ul>
-          </li> 
-        </ul>
       </div>
     </section>
-    <footer class="form-cantact">
+    
+    <footer class="form-cantact" id="contact">
       <div class="form-container">
         <h2>Contact me</h2>
         <p class="form-paragraph">
-          If you have an application you <br>
-          are interested in developing, a <br>
-          feature that you need built or a <br>
-          project that needs coding. I’d <br>
-          love to help with it
+          If you have an application you <br>are interested in 
+          developing, a <br>feature that you need built or a <br>
+          project that needs coding. I’d <br>love to help with it
         </p>
         <form action="https://formspree.io/f/mqkjggay" method="post">
           <input type="text" name="name" maxlength="30" placeholder="Queenter" required><br>
```

Comment:
- [ ] [OPTIONAL] Good job increasing the size of your contact form for accessibility and visibility but I think you should consider adding some padding to your form elements i.e. inputs especially to the left - seeing that inputs are collapsing to the edge making it difficult to read well.
- _Try to see what I'm talking about_ 👇

![image](https://user-images.githubusercontent.com/99715640/213873631-36b37e29-36e2-45c8-8c38-f11702ddfbc2.png)